autoptimize.php

```diff
@@ -61,8 +64,15 @@
 	$autoptimize_db_version=$autoptimize_version;
 }
 
+// Load translations from the languages directory.
```

Comment:
Tested without those lines, works as well (on condition the locale is not in `autoptimize/localization`, so no need to add that here.